src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -177,6 +177,16 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
     @Optional
     Long shmSize
 
+    /* 
+     * Automatically remove the container when the container's process exits.
+     *
+     * This has no effect if {@link #restartPolicy} is set.
+     * @since 3.6.0
```

Comment:
At the moment the next version is probably going to be `3.5.1` though I think we should bump the [existing changes](https://github.com/bmuschko/gradle-docker-plugin/milestone/70?closed=1) to 3.6.0. @cdancy Do you agree?

Reply:
Agreed.

Reply:
@cdancy I changed the milestone to 3.6.0.

Reply:
@cdancy This should be 3.6.2.

Reply:
On it